lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -6,14 +6,19 @@
 
 namespace Js
 {
-    bool JavascriptProxy::IsRevoked() const
+    bool JavascriptProxy::IsRevoked() const noexcept
     {
         return (target == nullptr);
```

Comment:
I think it's the case that if `target == nullptr` then `handler == nullptr` as well. If so, maybe add an `Assert(handler == nullptr)` to this method?